calculator/interfaces/Main.java

```diff
@@ -0,0 +1,87 @@
+package calculator.interfaces;
+import java.util.Scanner;
+import java.util.InputMismatchException;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String calculatorModel = "0";
+
+        closeProgram: 
+            while(true) {
```

Comment:
no need for that infinite loop, you are calling break closeProgram;  on exit anyway. Removing that while loop and calling just break would result in the same behavior.